packages/web/src/components/cell/createSuspendingCell.tsx

```diff
@@ -107,7 +107,10 @@ export function createSuspendingCell<
     const FailureComponent = ({ error, resetErrorBoundary }: FallbackProps) => {
       if (!Failure) {
         // So that it bubbles up to the nearest error boundary
-        throw error
+        if (error) {
+          throw error
+        }
+        throw new Error('Unreachable code: FailureComponent without a Failure')
```

Comment:
Throwing `undefined` would still have been an error that we'd have seen/heard of. I doubt anyone else has added something like `Unreachable code:` to our codebase before so it should be fairly easy to spot if this change results in an issue.